core/trino-main/src/main/java/io/trino/server/security/ResourceAccessType.java

```diff
@@ -49,15 +49,13 @@ public AccessType getAccessType(ResourceInfo resourceInfo)
             // check if the resource class has an access type declared for all methods
             accessType = resourceAccessTypeLoader.getAccessType(resourceInfo.getResourceClass());
             if (accessType.isPresent()) {
-                verifyNotTrinoResource(resourceInfo);
```

Comment:
@dain this prohibited @ResourceSecurity at the class level for io.trino Jax-RS resources. Was it on purpose?